datalad/distribution/create_sibling.py

```diff
@@ -202,6 +203,12 @@ def _create_dataset_sibling(
         shared = CreateSibling._get_ds_remote_shared_setting(
             delayed_super, name, ssh)
 
+    if group:
+        # Either it existed before or a new directory for a repo, set its
+        # group to a desired one if was provided
```

Comment:
I'm having trouble parsing this.

Reply:
I just wanted to say that the same command (chgrp) will be used regardless either it (directory/repository) already existed or we just created a new directory for it

Reply:
will try to make it more english since anyways need to push